### PR TITLE
Logo component moved from flow to type script.

### PR DESCRIPTION
**Pull request by DipakShrestha18**

### CODE REVIEW

File: src/components/Logo/Logo.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  className?: string;
+  data?: {
+    viewBox?: string;
+    fillRule?: 'nonzero' | 'evenodd' | 'inherit';
+    path?: string;
+  };
+};
+
+export const Logo = ({
+  title = '',
```
Comment (chriddell) [inline src/components/Logo/Logo.tsx]: Would remove default assignments (`= ''`) as they're unnecessary.

File: src/components/Logo/svg/wellcome-logo-large.svg [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="80px" height="80px" viewBox="0 0 80 80" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 54.1 (76490) - https://sketchapp.com -->
+    <title>wellcome-logo-large</title>
+    <desc>Created with Sketch.</desc>
```
Comment (chriddell) [inline src/components/Logo/svg/wellcome-logo-large.svg]: Meta is unnecessary and adds bloat, should remove.

File: src/components/Logo/svg/wellcome-logo-large.svg [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    <g id="wellcome-logo-large" stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
```
Comment (chriddell) [inline src/components/Logo/svg/wellcome-logo-large.svg]: Remove the `id` attribute.

File: src/components/Logo/svg/wellcome-logo-medium.svg [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="60px" height="60px" viewBox="0 0 60 60" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 54.1 (76490) - https://sketchapp.com -->
+    <title>wellcome-logo-medium</title>
+    <desc>Created with Sketch.</desc>
```
Comment (chriddell) [inline src/components/Logo/svg/wellcome-logo-medium.svg]: Same again, remove meta.

File: src/components/Logo/svg/wellcome-logo-medium.svg [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+    <g id="wellcome-logo-medium" stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
```
Comment (chriddell) [inline src/components/Logo/svg/wellcome-logo-medium.svg]: ...and `id`

File: src/components/Logo/svg/wellcome-logo-small.svg [hunk 6]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="146px" height="44px" viewBox="0 0 146 44" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 54.1 (76490) - https://sketchapp.com -->
+    <title>wellcome-logo-small</title>
+    <desc>Created with Sketch.</desc>
+    <g id="wellcome-logo-small" stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
```
Comment (chriddell) [inline src/components/Logo/svg/wellcome-logo-small.svg]: Same again - remove meta and id

**Review by chriddell — CHANGES_REQUESTED** [review]
Some minor tidy-ups needed but nothing major - we may also need to convert the `.svg`s to React components in the future, but should be fine for now!

File: src/components/Logo/svg/wellcome-logo-large.svg (hunk 2 above)
Comment (creido-welly) [reply src/components/Logo/svg/wellcome-logo-large.svg]: 👍 +1

recommend you run these svg files through this tool to strip out all unnecessary bloat
https://jakearchibald.github.io/svgomg/

File: src/components/Logo/Logo.tsx (hunk 1 above)
Comment (DipakShrestha18) [reply src/components/Logo/Logo.tsx]: I've made all the changes as requested.

Comment (chriddell) [reply src/components/Logo/Logo.tsx]: Cool will review/approve once pushed

Comment (creido-welly) [reply src/components/Logo/Logo.tsx]: @DipakShrestha18 please can you try pushing your changes again. I can still see the default assignments

Comment (DipakShrestha18) [reply src/components/Logo/Logo.tsx]: done.

File: src/components/Logo/svg/wellcome-logo-large.svg [hunk 7]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="80px" height="80px" viewBox="0 0 80 80" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 54.1 (76490) - https://sketchapp.com -->
```
Comment (chriddell) [inline src/components/Logo/svg/wellcome-logo-large.svg:3]: Remove the comment too (sorry, that's my fault, should've specified that in previous comment...)

File: src/components/Logo/svg/wellcome-logo-small.svg [hunk 8]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="146px" height="44px" viewBox="0 0 146 44" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 54.1 (76490) - https://sketchapp.com -->
```
Comment (chriddell) [inline src/components/Logo/svg/wellcome-logo-small.svg:3]: Same again, remove comment.

**Review by chriddell — CHANGES_REQUESTED** [review]

**Review by creido-welly — APPROVED** [review]

**Pull request merged**